mobility_games/games/wav_musical_cane.py

```diff
@@ -132,16 +136,25 @@ def tag_array(self, msg):
                 self.tag_list.append((x, y, z, tag_id))
                 self.id_list.append(tag_id)
 
-
     def config_callback(self, config, level):
         self.mode = config["mode"]
         self.sweep_tolerance = config["sweep_tolerance"]
         self.sweep_range = config["sweep_range"]
+        self.count_sweeps = config["count_sweeps"]
+
+        # set reward checkpoints
+        self.reward_at[10] = config["reward_at_10"]
+        self.reward_at[20] = config["reward_at_20"]
+        self.reward_at[50] = config["reward_at_50"]
+        self.reward_at[100] = config["reward_at_100"]
+        self.reward_at[250] = config["reward_at_250"]
+        self.reward_at[500] = config["reward_at_500"]
+        self.reward_at[1000] = config["reward_at_1000"]
```

Comment:
Might be a little more concise and modular in a for loop, in case we wanted to change which reward numbers we wanted to use?

``` 
for num in [10, 20, 50, 100, 250, 500, 1000]:
    self.reward_at[num] = config["reward_at_%s" % num] 
```